from app.core.exceptions.company.company import CompanyNotFoundError